binding/web/src/cobra.ts

```diff
@@ -351,11 +354,13 @@ export class Cobra {
 
     const status = await pv_cobra_init(accessKeyAddress, objectAddressAddress);
```

Comment:
what about `_pvCobraProcess`?

Reply:
for engines who use device activation, we don't need to add them since they don't use js functions